packages/react-query/package.json

```diff
@@ -10,10 +10,17 @@
     "type": "github",
     "url": "https://github.com/sponsors/tannerlinsley"
   },
-  "module": "build/esm/index.js",
-  "main": "build/cjs/react-query/src/index.js",
+  "module": "build/lib/index.mjs",
+  "main": "build/lib/index.js",
   "browser": "build/umd/index.production.js",
   "types": "build/types/packages/react-query/src/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./build/lib/index.mjs",
+      "default": "./build/lib/index.js"
+    },
+    "./package.json": "./package.json"
+  },
   "sideEffects": [
```

Comment:
the files listed here don't exist - can we try to point them towards existing files?
right now, I think `setBatchedUpdatesFn` is treeshaked away because it's unused and the sideEffects listings are wrong

Reply:
Fixed